contrib/OpenInExplorer.lua

```diff
@@ -104,8 +104,13 @@ end
 -- GUI --
 if proper_install then
   dt.gui.libs.image.register_action(
-    _("show in file explorer"),
+    _("Show in file explorer"),

```

Comment:
The darktable standard is that all actions start with lower case letters.  Look at the selected images module and notice all the actions are named that way.

Reply:
You are right. I've never managed to integrate this .... it's such a weird convention.
I'll correct this ...
Thanks @wpferguson

Reply:
OK, I'm lost .... I don't know how to change this.
I'll start from scratch :facepalm: